virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -194,6 +216,7 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-28723 Release note: PREVIEW
 
 //CNV-28944 Release note: Preview Cluster level eviction strategy change
+* You can now configure a VM eviction strategy for the xref:../../virt/nodes/virt-node-maintenance.adoc#virt-configuring-cluster-eviction-strategy-cli_virt-node-maintenance[entire cluster].
```

Comment:
Would be very helpful to embed a link for "eviction strategy"  too, using this link, https://66553--docspreview.netlify.app/openshift-enterprise/latest/virt/nodes/virt-node-maintenance#eviction-strategies

As there are many new options available to be set starting 4.14.

Reply:
I've added the link, @kbidarkar, and I'm now sending it for per review. Thanks very much.